src/components/atoms/Alert/index.js

```diff
@@ -3,11 +3,11 @@ import Snackbar from "@mui/material/Snackbar";
 import MuiAlert from "@mui/material/Alert";
 import { useAuth } from "context/getAuth";
 
-export const Alert = ({ variant = AlertTypes.error }) => {
-  const { isAlertOpen, alertText } = useAuth();
+export const Alert = ({ variant = AlertTypes.error, open }) => {
+  const { alertText } = useAuth();
```

Comment:
czemu `textAlert` jest w `auth`?

Reply:
ponieważ to komponent (alert) został stworzony
 przez Ciebie i tak zrobiłeś :d wiec z góry przyjąłem ze forma ta jest poprawna :D